src/ray/gcs/gcs_server/gcs_worker_manager.cc

```diff
@@ -94,17 +94,15 @@ void GcsWorkerManager::HandleReportWorkerFailure(
     int count = 0;
     if (request.worker_failure().exit_type() == rpc::WorkerExitType::SYSTEM_ERROR) {
       worker_crash_system_error_count_ += 1;
-      key = "extra_usage_tag_worker_crash_system_error";
+      key = "worker_crash_system_error";
       count = worker_crash_system_error_count_;
     } else if (request.worker_failure().exit_type() ==
                rpc::WorkerExitType::NODE_OUT_OF_MEMORY) {
       worker_crash_oom_count_ += 1;
-      key = "extra_usage_tag_worker_crash_oom";
+      key = "worker_crash_oom";
```

Comment:
ideally we would move those keys into proto files, but we can do it in my follow up PRs

Reply:
Yea sounds good. Let's do it in your PR.